new-index-file

```diff
@@ -0,0 +1 @@
+
```

Comment:
<h1>
      🌤
      <br />
      Currently 21° in Tokyo
    </h1>
    <h2>13° / <strong>23°</strong></h2>
    <ul>
      <li>
        <h3>🌧Tomorrow</h3>
        <p>10° / <strong>22°</strong></p>
      </li>
      <li>
        <h3>🌥 Saturday</h3>
        <p>15° / <strong>17°</strong></p>
      </li>
      <li>
        <h3>☀️ Sunday</h3>
        <p>25° / <strong>28°</strong></p>
      </li>
    </ul>

    <button>Change city</button>
    <p>Coded by Pamela Gwala</p>